ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/core/PrivacyParameters.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.ethereum.core;
+
+import java.io.File;
+import java.net.URI;
+
+public class PrivacyParameters {
+
+  private static final String ORION_URL = "http://localhost:8888";
+  public static final URI DEFAULT_ORION_URL = URI.create(ORION_URL);
+
+  private boolean enabled;
+  private String url;
+  private File publicKey;
+
+  public File getPublicKey() {
+    return publicKey;
+  }
+
+  public void setPublicKey(final File publicKey) {
+    this.publicKey = publicKey;
+  }
+
+  public static PrivacyParameters createDefault() {
```

Comment:
I wonder if we should name this differently. Maybe `noPrivacy()` or something to indicate that the default is using Pantheon without any privacy features. Thoughts?

Reply:
Yes, that makes sense. It makes it clear that the default we use has privacy disabled.
Will update the same.